src/eduid_webapp/group_management/schemas.py

```diff
@@ -71,3 +104,39 @@ class GroupCreateRequestSchema(EduidSchema, CSRFRequestMixin):
 class GroupDeleteRequestSchema(EduidSchema, CSRFRequestMixin):
 
     identifier = fields.UUID(required=True)
+
+
+class GroupRemoveUserRequestSchema(EduidSchema, CSRFRequestMixin):
+
+    group_identifier = fields.UUID(required=True)
+    user_identifier = fields.UUID(required=True)
+    role = fields.Str(required=True, validate=validate_role)
+
+
+class GroupInviteRequestSchema(EduidSchema, CSRFRequestMixin):
+
+    identifier = fields.UUID(required=True)
+    email_address = fields.Email(required=True, validate=[validate_email])
+    role = fields.Str(required=True, validate=[validate_role])
```

Comment:
how come validate is a list of validate_role here, but not a list in GroupRemoveUserRequestSchema above?